flink-java/src/main/java/org/apache/flink/api/java/operators/SingleInputUdfOperator.java

```diff
@@ -114,41 +114,41 @@ public O withBroadcastSet(DataSet<?> data, String name) {
 	}
 
 	/**
-	 * <p>
-	 * Adds semantic information about forwarded fields of the user-defined function.
+	 *
+	 * <p>Adds semantic information about forwarded fields of the user-defined function.
```

Comment:
`<p>` should be removed as  this is the first sentence. Empty line above should be removed as well.